sdk/servicebus/Azure.Messaging.ServiceBus/src/Administration/ServiceBusAdministrationClient.cs

```diff
@@ -207,6 +207,34 @@ public ServiceBusAdministrationClient(string connectionString)
         ///
         /// <remarks>Works with any claim (Send/Listen/Manage).</remarks>
         /// <returns><see cref="NamespaceProperties"/> containing namespace information.</returns>
+        ///
+        /// <exception cref="UnauthorizedAccessException">No sufficient permission to perform this operation.
+        /// You should check to ensure that your <see cref="ServiceBusAdministrationClient"/> has the necessary
+        /// claims to perform this operation.
+        /// <seealso href="https://docs.microsoft.com/azure/service-bus-messaging/service-bus-sas#rights-required-for-service-bus-operations"/>
```

Comment:
I'm not entirely sure this will render in MS Docs embedded in an `<exception>` tag.  There were some odd rules around `<seealso>` .... we should double-check the day after we publish to see if we need to adjust after.